src/components/Blocks/Hero/Hero.tsx

```diff
@@ -2,14 +2,8 @@ import React, { useEffect } from 'react';
 import styles from './Hero.module.scss';
 
 export function Hero(props) {
-    useEffect(() => {
-        const container: HTMLElement | null = document.getElementById('container');
-        if (props.bool == true) {
-            container.classList.add(styles['reverse']);
-        }
-    });
     return (
-        <div id="container" className={styles.container}>
+        <div id="container" className={props.bool ? styles.reversecontainer : styles.container}>
```

Comment:
What does props.bool mean here?

Reply:
It's the boolean that determines if the hero block is right hand or left hand. I've renamed it to swapSides.